### DISCUSSION TITLE
[Suggestion] Add an option "Forget opened files on exit"

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2918] [Features Request & Ideas | 特性请求 & 头脑风暴]
I don't like an idea of piling up multiple text files in text editors. I wish you could turn it off

**Comment by meiyixiang** [comment]
What do you expect? Can be presented in a separate window?

**Reply by BLumia** [reply]
I think they means they prefer open a new session every time user launch deepin-editor, instead of open the last files user didn't closed.

我觉得意思是说希望可以每次打开的时候可以开新的编辑器实例，不恢复打开之前未主动关闭的标签。

**Comment by meiyixiang** [comment]
The positioning of the text editor product is a little strange. I will record your problems and feed back to the product so as to optimize the interaction and operation logic of the product
